### PR TITLE
Fixes #10 (structure for repo)

### LINKED ISSUE #10: Set up a structure for the repo
The template structure is not so proper. A proper structure makes it easy to find the files and code.
Suggested actions:
1. Create a static folder at the root with `css`,`js` and `img` folder inside it.
2. All the `js`,`css` and `img` should be moved to their respective folders.
3. `index.html` stays in the root of the repository.
4. Remove `gulp.js`, `package.json` and `package-lock.json`.

### Please note that changing folder names and file path might break their import in the html file, make sure to fix those also in the PR.

### PR DESCRIPTION
**Pull request by Archies11**
I have added all the css, js files to respective folders.

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@Archies11 Kindly make the respective changes in index.html file so that the code doesn't break

File: static/css/temp [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Tlazypanda) [inline static/css/temp]: @Archies11 Can you explain these temporary files added?

**Comment by Tlazypanda** [comment]
@Archies11 Also provide a gh pages link :sweat_smile:

File: static/css/temp (hunk 1 above)
Comment (Archies11) [reply static/css/temp]: I added these files while creating the folders as empty folders cant be created. i will delete them right now and provide a link gh pages link as well

**Comment by Archies11** [comment]
I have deleted the files.
Link: https://archies11.github.io/GirlsDoCode/

Sorry for forgetting to add the same.

**Comment by Tlazypanda** [comment]
@Archies11 Hey this is opensource don't be sorry 😅 LGTM! 😄

**Pull request merged**